README.md

```diff
@@ -79,6 +79,7 @@ Priority order:
 | resources.syncBatch                           |        4096        |    No    |How many resources to sync in one go.
 | resources.cacheExpiration                     |       300000       |    No    |Expiration in milliseconds for synced resources in Redis.
 | resources.compressionMinSize                  |        256         |    No    |Compress a resource with gzip if its size in bytes more or equal to this value.
+| resources.redisPrefix                         |      resource      |    No    |Prefix name is used to form Redis key for resource's queue and topic.
```

Comment:
Default should be used from storage, otherwise it is breaking change

Reply:
it's not breaking change. the default value is taken from settings.json

Reply:
Blob prefix should be the default in this case. People already set some values to blob prefix. If they run the old config it won't work.

Reply:
discussed privately. Let blob storage prefix is used to make a different names for topic and queue in Redis